crates/ruff/src/importer/mod.rs

```diff
@@ -75,6 +75,30 @@ impl<'a> Importer<'a> {
         }
     }
 
+    /// Move an existing import to the top-level, thereby making it available at runtime.
+    ///
+    /// If there are no existing imports, the new import will be added at the top
+    /// of the file. Otherwise, it will be added after the most recent top-level
+    /// import statement.
+    pub(crate) fn to_runtime_import(&self, import: &StmtImport, at: TextSize) -> Result<Edit> {
```

Comment:
Nit: I would expect a `to_runtime_import` method to return a `RuntimeImport`. How about `to_runtime_import_edit()`